equipment_handler/source/adc_handler.c

```diff
@@ -17,14 +17,14 @@
  * @date    2020-06-15
  */
 
-#include <ex2_hal/ex2_hyperion_solar_panel_software/equipment_handler/include/adc_handler.h>
```

Comment:
I think Robert commented a while ago that the includes were a little off of standard. Not sure if there is anything wrong with this, but just bringing it up.

Reply:
Yes I have changed it into `#include "adc_handler.h"`